src/memote.jl

```diff
@@ -321,3 +321,73 @@ function is_consistent(
     value.(m)
     termination_status(opt_model) == OPTIMAL
 end
+
+
+
+"""
+Checks for the presence of metabolite annotations.
+Returns a vector of all metabolites without any annotations.
+"""
+
+function all_unannotated_metabolites(model::StandardModel)
```

Comment:
Let's just make this `::MetabolitModel` and add a TODO comment to change it to `AbstractMetabolicModel` once COBREXA 2.0 releases

Reply:
This seems fixed to me